src/Tribe/Utils/Lazy_String.php

```diff
@@ -71,7 +71,12 @@ public function __construct( callable $callback, $escape_callback = 'esc_html' )
 	 */
 	public function __toString() {
 		if ( null === $this->string ) {
-			$this->string = call_user_func( $this->value_callback );
+			$value =  call_user_func( $this->value_callback );
+			if ( ! $value instanceof \Generator ) {
```

Comment:
Why `Generator`?

Reply:
Because if we use `yield` it will return that.